src/frontend/components/UI/DialogHandler/components/MessageBoxModal/index.tsx

```diff
@@ -17,8 +17,24 @@ interface MessageBoxModalProps {
   type: DialogType
 }
 
+function decodeHTML(html: string) {
```

Comment:
would be nice to add a comment on this for the future, I think it makes sense now because I can see the PR but it's kinda confusing without context